src/lib/PEFile.ts

```diff
@@ -54,8 +54,8 @@ export class PEFile {
 
         const optionalHeaderOffset = ntHeadersOffset + 4 + this.FILE_HEADER.sizeof();
         const sectionsOffset = optionalHeaderOffset + this.FILE_HEADER.SizeOfOptionalHeader;
-
-        this.OPTIONAL_HEADER = new OptionalHeader(optionalHeaderOffset).unpack(
+        const x64 = data[optionalHeaderOffset] === 0x0b && data[optionalHeaderOffset + 1] === 0x02;
+        this.OPTIONAL_HEADER = new OptionalHeader(optionalHeaderOffset, x64).unpack(
```

Comment:
Instead of using **raw** values, utilize `Unpack.unpack` method and convert the **Magic** to its valid data type and then check its value (which is `0x20b` for **64-bit** PE file)
```suggestion
        const is64bit = Unpack.unpack("<H", data, optionalHeaderOffset)[0] === 0x20b;
        this.OPTIONAL_HEADER = new OptionalHeader(optionalHeaderOffset, is64bit).unpack(
```